packages/core/README.md

```diff
@@ -0,0 +1,22 @@
+# TV Kitchen Appliance Core
+
+This repository contains core classes and constants for use by TV Kitchen Appliances and code that interacts with TV Kitchen Appliances.
```

Comment:
Now that this is a monorepo and this README specific to the core appliance, the language here (and below) should change from "This repository" to "This package", shouldn't it?

```suggestion
This package contains core classes and constants for use by TV Kitchen Appliances and code that interacts with TV Kitchen Appliances.
```